airbyte-webapp/src/pages/ConnectorBuilderPage/ConnectorBuilderPage.module.scss

```diff
@@ -1,16 +1,23 @@
 @use "scss/colors";
 @use "scss/variables";
+@use "scss/mixins";
 
 .leftPanel {
   overflow: hidden;
 }
 
 .rightPanel {
-  border-top-left-radius: variables.$border-radius-lg;
-  border-bottom-left-radius: variables.$border-radius-lg;
+  @include mixins.left-shadow;
+
+  border-top-left-radius: variables.$border-radius-md;
```

Comment:
Why are there border radiuses on the right panel? The left sidebar doesn't have them and the setup guide that's shown in a very similar fashion in the connector form doesn't have them either.

Reply:
I was trying to match the design here: https://www.figma.com/file/F1ljvrJerLoVbNvtejBWid/LOW-CODE-CONNECTOR-BUILDER?node-id=2643%3A109796&t=VcXAAc8881mHE2Rx-0

Personally I like the border radiuses because they make the right panel feel more like a slide-out view, which differentiates it a bit from the left sidebar since the size of that one cannot be adjusted / it is static.

I wouldn't be opposed to adding the border radiuses to the documentation panel in the setup guide as well

Reply:
> I wouldn't be opposed to adding the border radiuses to the documentation panel in the setup guide as well

good idea, sounds great!